dex/src/main/scala/com/wavesplatform/dex/model/MatchTimestamp.scala

```diff
@@ -0,0 +1,9 @@
+package com.wavesplatform.dex.model
+
+object MatchTimestamp {
```

Comment:
probably we should add a comment here that it is used for making exchange transactions with strictly increasing timestamp

Reply:
It doesn't increase timestamp, just choose: use old logic (return takerOrderAddedTs), or find `max(takerOrderAddedTs, orderBookTs)`, but this piece of code looks so simple, so I don't think that comment is needed here